drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -441,6 +446,23 @@ function DragAndDropTemplates(configuration) {
         )
     };
 
+    var forwardKeyboardHelpButtonTemplate = function(ctx) {
+        return h(
+            'button.unbutton.btn-default.btn-link.'+"keyboard-help-button",
+            {attributes: {tabindex: 0}},
```

Comment:
@cahrens according to [W3C spec](https://www.w3.org/TR/html5/editing.html#sequential-focus-navigation-and-the-tabindex-attribute), setting it on a `button` element is not required, as by spec buttons are among the elements that have the tabindex focus flag set (but "modulo platform conventions"). On the other hand, setting `tabindex: 0` just sets that flag.

It is safe both to keep it or remove it, but keeping it has a slight advantages of:
1. No need to retest it now.
2. No need to redeploy sandbox.
3. In future if the type of element changes (i.e. button => span, e.g. for styling purposes) it will protect from button being inaccessible using keyboard.